README.md

```diff
@@ -13,13 +13,14 @@ git clone https://github.com/biocore/metagenomics_pooling_notebook.git
 Create a Python3 Conda environment in which to run the notebook:
 
 ```bash
-conda create -n pooling_nb 'python>=3.6' scipy numpy matplotlib pandas
+conda create -n metapool 'python=3.9' scikit-learn pandas numpy scipy nose pep8 flake8 matplotlib jupyter notebook 'seaborn>=0.7.1' pip openpyxl
```

Comment:
Can you also change the README to reflect this change?

Reply:
Actually this is the readme file that's being changed.